ocp_resources/resource.py

```diff
@@ -654,13 +655,21 @@ def update_replace(self, resource_dict):
         self.api.replace(body=resource_dict, name=self.name, namespace=self.namespace)
 
     @staticmethod
-    def _retry_cluster_exceptions(func):
+    def _retry_cluster_exceptions(
```

Comment:
```suggestion
    def retry_cluster_exceptions(
```
this is no longer a private function

Reply:
ok. I will update it and ensure the associated calls are changed as well.